### PR TITLE
Fix buttons background for devices with <=17 API

### PR DESCRIPTION
**Pull request by johncodeos**
For devices with  <=17 API :
**Before**
screenshot_1512492011
**After**
screenshot_1512582622

I know Pull requests from external contributors are not currently being accepted but I'll let it here maybe in the future you can add it if you want

### CODE REVIEW

File: amplify/src/main/java/com/github/stkent/amplify/prompt/DefaultLayoutQuestionView.java [hunk 1]
```diff
@@ -169,11 +170,20 @@ private void setButtonViewBackground(
         final Drawable buttonBackgroundDrawable
                 = getButtonBackgroundDrawable(fillColor, borderColor, borderWidthPx, cornerRadiusPx);
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+
+        if (android.os.Build.VERSION.SDK_INT <= 17) {
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/prompt/DefaultLayoutQuestionView.java]: Is there a reference you can point me to to explain why this change is necessary? I see the issue on my API level 16 AVD, but I'm struggling to find and reference to changes in API levels 16/17/18 that explain why this workaround is necessary.

Thanks!

**Comment by johncodeos** [comment]
API 17 is too old these days...😄 I don't even support it in my apps anymore. You don't need to fix it anymore.

**Comment by stkent** [comment]
Haha thanks for the update! The PR got closed because I switched primary branch name from master to main. I'll make an issue to lower the minimum support SDK version to 21 since that seems pretty typical these days.
